### ISSUE TITLE
Why is All the Steps Content Not Displaying 

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#12]
Why is all the **Steps** content not displaying when I go to those pages?

It makes no sense to have to open all the content sections up to see what's on the page.

I thought the purpose of the application was so people could find content, read it, and then take an action of some sort. If the content is closed then they have to take the effort to open each section. If they don't feel like opening the sections they will not read or get the information you want to find.

Plus, you only can open the content by tapping on the down arrow instead of the words and the down arrow. i will open a separate issue for that.

**NOTE** - My testing is being done on our iPad Air 2 (iOS 10.3.3) because our iPod Touch 5th Generation (iOS 9.3.5) opens the application but displays a blank screen.

**Comment by adriennerlong** [comment]
We wanted the steps pages to be separated into Learning Objectives, Key Terms, Topics, and Further Resources. The topics are laid out because they're the meat of each step. Not all of the steps will be useful for every user, so this way they can quickly glance at the step's topics and decide whether they want to dive deeper into the Step. Please close out on this issue.

**Comment by jfc3-dol** [comment]
@adriennerlong,

If all I see if different sections on a page with no content displayed as a user I would wonder if it is there or even useful.

It wouldn't be so bad if there was at least some kind of starting paragraph about the step and not an empty screen besides the different section titles.

Not the most user friendly experience.

**Comment by jfc3-dol** [comment]
Application screen is not useful to me when the screen only contains headings. Not sure at a quick glance is if things are closed or there is no content. As a user, I would like to see the content and not have to open each section to see it.

See attached screenshot.

![complychain - closed content in the steps section of application](https://user-images.githubusercontent.com/10423487/29888474-16e401a8-8d8f-11e7-8c57-500ba5d5ce24.png)

**Comment by adriennerlong** [comment]
We've asked the developers to keep the Learning Objectives, Key Terms, and Further Resources tabs un-expanded and to expand the Topics tab. This way, users will be able to see what each step is about. 

I like your idea of having an introductory paragraph for each topic, but unfortunately we just don't have time to write them and get the language cleared. We'll think about implementing this is version 2.0.

**Comment by jfc3-dol** [comment]
@adriennerlong,

I think they all should be open so people can see the content and read what they want even if they have to scroll.

**Comment by jfc3-dol** [comment]
@adriennerlong,

At this point all sections are closed up when going to the page so people could think there is no content.

**Comment by jfc3-dol** [comment]
Same applies to our Android device.

**Comment by adriennerlong** [comment]
From ICF: Fixed, the Topics section no longer expands or collapses.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Okay, by having the other sections closed users are more likely not to open them and read the content that is hidden.

**Comment by jfc3-dol** [comment]
As stated previously with having the content closed to iOS, Android, and the web versions makes it less likely that the user will look for or find the content you want them to read or find.

**Comment by jfc3-dol** [comment]
Closing issue because ILAB does not plan to show all content when a person goes to those pages for all three platforms (web, iOS, and Android).
